gulpfile.js

```diff
@@ -356,8 +356,8 @@ gulp.task('build', gulp.series(
     'build:hexo',
     'copy:robots.txt',
     // 'generate-service-worker',
-    'compress:zopfli',
-    'compress:brotli',
+    //'compress:zopfli',
```

Comment:
nit: Can you add a `TODO` or a comment that we are disabling this until github supports compressed assets.